cli/util/file_watcher.rs

```diff
@@ -109,26 +108,85 @@ fn create_print_after_restart_fn(clear_screen: bool) -> impl Fn() {
   }
 }
 
+// TODO(bartlomieju): this is a poor name; change it
+/// And interface to interact with Deno's CLI file watcher.
+pub struct WatcherInterface {
```

Comment:
@dsherret any suggestions for the name of this struct?

Reply:
ChangeNotifierInterface ?

Reply:
Maybe like `WatcherChannel`. It's bidirectional communication between inside and outside the watched operation